code/api/src/setup/database.js

```diff
@@ -16,19 +16,15 @@ const connection = new Sequelize(databaseConfigEnv.database, databaseConfigEnv.u
 })
 
 // Test connection
-console.info('SETUP - Connecting database...')
+// console.info('SETUP - Connecting database...')
 
-connection
-  .authenticate()
-  .then(() => {
-    console.info('INFO - Database connected.')
-  })
-  .catch(err => {
-    console.error('ERROR - Unable to connect to the database:', err)
-  })
+// connection
```

Comment:
N ice catch on this, I had an issue with this on a branch I was working on as well.